### DISCUSSION TITLE
Do I need a collection for non-iterating things? I have a homepage with 10 text blocks

### DISCUSSION THREAD

**Discussion by notflip** [statamic/cms#7012] [General]
I'm wondering what the statamic-way is to have my client be able to edit text blocks on the homepage, there will only be one or two pages.

**Comment by nadinengland** [comment]
You could make a Global Set which is purely for your CMSable text blocks, i.e. `page_settings` and stick your text blocks on that on that.

I don't believe there is a way to make collection a "singleton", but the Global Set method would give you the same results!

**Comment by ryanmitchell** [comment]
Personally I would use a replicator field for that. Have a look at the peak starter kit - you can get a lot of inspiration from how it is structured.

**Reply by jasonvarga** [reply]
Yes, you can create a one-off `home` blueprint in your `pages` collection, assign it to the home entry by adding `blueprint: home`, and mark the blueprint as hidden so you can't use it for other pages.
